hack/verify-manifests.sh

```diff
@@ -6,6 +6,6 @@ trap "test -d $tmpdir && rm -rf $tmpdir" EXIT
 echo "Backup config directory to $tmpdir first"
 cp -r config $tmpdir/config
 
-make update
+make rbac manifests
```

Comment:
This makes filename misleading. It must be `hack/verify-rbac-and-manifests.sh` then.

Reply:
I split this into `verify-rbac.sh` and `verify-manifests.sh` instead, so they each have a clear function, and we could extend one without affecting the other in the future.